resources/content/db/changelog.xml

```diff
@@ -33,5 +33,6 @@
     <include file="db/core-026.xml"/>
     <include file="db/core-027.xml"/>
     <include file="db/core-028.xml"/>
+    <include file="db/errorhandling.xml"/>
```

Comment:
I'll make the correct change when we are ready to merge.  I didn't want to deal with merge-conflicts later in case someone else is going to check in db changes.